Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f030a3abe50>, 'Connection to ipinfo.io timed out. (connect timeout=5.0)'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f030c517750>, 'Connection to ipinfo.io timed out. (connect timeout=5.0)'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1b6284e10>: Failed to establish a new connection: [Errno 113] No route to host'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1b4113290>: Failed to establish a new connection: [Errno 113] No route to host'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1b4fef5d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1b4110290>: Failed to establish a new connection: [Errno -2] Name or service not known'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedd64c4ed0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedd412f750>: Failed to establish a new connection: [Errno -2] Name or service not known'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedd412e050>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedd412de10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc8e31773d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc8e164b550>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc8e1649cd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc8e1649ad0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedff778c50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedfdc51d10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedfdc513d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedfdc512d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fda12468f50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fda11503210>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fda136d5690>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fda11503790>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc75f9950>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc66df6d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc66dd990>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc66deb90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb6b8c1c450>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Status code Unknown from http://ipinfo.io/json: ERROR - HTTPConnectionPool(host='ipinfo.io', port=80): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb6b88f3710>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
